app/views/providers/addresses/_form.html.erb

```diff
@@ -39,13 +40,14 @@
 
     <% group_error_class = @form.errors[:address_line_one].any? ? 'govuk-form-group--error' : '' %>
     <div class="govuk-form-group <%= group_error_class %>">
-      <%= form.govuk_text_field :address_line_one , 'aria-label': 'Building and street line 1 of 2' %>
-      <%= form.govuk_text_field :address_line_two, field_with_error: :address_line_one, 'aria-label': 'Building and street line 2 of 2' %>
+      <%= form.govuk_text_field :address_line_one , label: {text: "Building and street"}, 'aria-label': 'Building and street line 1 of 2' %>
+      <%= form.govuk_text_field :address_line_two, label: {text: "Building and street", hidden: true},
+                                'aria-label': 'Building and street line 1 of 2', field_with_error: :address_line_one %>
```

Comment:
Should this be 'Building and street line 2 of 2',  not 1 of 2?

Reply:
oh yes good spot